webapp-django/crashstats/crashstats/models.py

```diff
@@ -1009,6 +1009,66 @@ def get(self, *args, **kwargs):
         }
 
 
+class VersionString(SocorroMiddleware):
+    # NOTE(willkg): This is implemented with a Django model.
+
+    # Set to a short cache time because it's just a db lookup. Making it non-0
+    # is to prevent the stampeding herd on this endpoint alone when exposed in
+    # the API.
+    cache_seconds = 2 * 60  # 2 minutes only
```

Comment:
Currently, this is used by the processor and it does its own caching. I want to avoid multiple layers of caching because it gets harder to reason about. Therefore, I made this 2 minutes.